File: strongNum.js
function strong(n) {
    let arr = String(n).split('');
    let sum = 0;
    for (let i = 0; i<arr.length; i++){
      sum += fact(Number(arr[i]))
    }
    return sum == n ? "STRONG!!!!" : "Not Strong !!";
  }
  
  function fact(num){
   let result = num;
    if (num === 0 || num === 1) 
      return 1; 
    while (num > 1){
      num--;
      result = result * num;
    }
    return result;
  }

File: strongNum.py
def strong_num(number):
    num = [int(i) for i in str(number)]
    sum = 0
    for x in num:
        sum += fact(x)
    
    if sum == number:
        return "STRONG!!!!"
    
    else:
        return "Not Strong !!"



def fact(num):
    result = num
    if num == 0 or num == 1:
        return 1
    while num > 1:
        num = num - 1
        result = result * num
        
    return result